build-bot/buildbot_setup.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+### Reading the input token for docker install
+if [ "$1" != "" ]; then
+    TOKEN=$1
+else
+    echo "Enter GitHub token as positional argument"
+fi
+
+### Base packages installation
+sudo apt-get update && upgrade
+sudo apt-get install openssh-server
```

Comment:
Can you lump these into a single `apt-get` call?

Reply:
Done